src/index.js

```diff
@@ -49,19 +49,22 @@ function parse(model, response, options = {}) {
 export default Bookshelf => {
   const Model = Bookshelf.Model.prototype;
   const client = Bookshelf.knex.client.config.client;
-  const parseOnFetch = client === 'sqlite' || client === 'sqlite3';
+  const handleOnSaving = client !== 'mysql';
```

Comment:
Sure thing.  Disabling the stringify on save results in the following:
```
{ method: 'insert',
  options: {},
  timeout: false,
  cancelOnTimeout: false,
  bindings: 
   [ { street: '1234 Road Ave',
       city: 'Town',
       zip: 32309,
       someOtherField: 'data' },
     Thu Jan 19 2017 09:39:17 GMT-0500 (EST),
     'test@example.com',
     '72261b8c-752f-4196-96b7-6b115d921f21',
     'Test User',
     'test-realm',
     Thu Jan 19 2017 09:39:17 GMT-0500 (EST) ],
  __knexQueryUid: '9ed6cc46-ec48-465a-9df2-0d7b1299df9c',
  sql: 'insert into `users` (`address`, `created_at`, `email`, `id`, `name`, `phoneNumber`, `realm`, `updated_at`) values (?, ?, ?, ?, ?, DEFAULT, ?, ?)' }
Error: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1
```
`address` is the field that should be stringified.

Enabling stringify on save works great:
```
{ method: 'insert',
  options: {},
  timeout: false,
  cancelOnTimeout: false,
  bindings: 
   [ '{"street":"1234 Road Ave","city":"Town","zip":32309,"someOtherField":"data"}',
     Thu Jan 19 2017 09:45:29 GMT-0500 (EST),
     'test@example.com',
     'dc608afb-3731-4d31-9a07-73cb97cee673',
     'Test User',
     'test-realm',
     Thu Jan 19 2017 09:45:29 GMT-0500 (EST) ],
  __knexQueryUid: '4b5508eb-c3a0-4755-b8f0-6b053d98ec56',
  sql: 'insert into `users` (`address`, `created_at`, `email`, `id`, `name`, `phoneNumber`, `realm`, `updated_at`) values (?, ?, ?, ?, ?, DEFAULT, ?, ?)' }
```
The `address` gets stringified and appropriately inserted into the TEXT field in the mysql db.